package.json

```diff
@@ -27,6 +27,13 @@
     "@babel/preset-react": "7.0.0-beta.44",
     "@babel/preset-stage-2": "7.0.0-beta.44",
     "@babel/runtime": "7.0.0-beta.44",
+    "@wordpress/a11y": "1.0.7",
```

Comment:
In the recent version of Gutenberg, they all are included in Gutenberg. No need to pull them in from npm.